lib/ViewModels/TerriaViewer.ts

```diff
@@ -21,8 +21,19 @@ import NoViewer from "../Models/NoViewer";
 import Terria from "../Models/Terria";
 import ViewerMode from "../Models/ViewerMode";
 
-// A class that deals with initialising, destroying and switching between viewers
-// Each map-view should have it's own TerriaViewer
+// Async loading of Leaflet and Cesium
+
+const leafletFromPromise = computed(
+  () =>
+    fromPromise(import("../Models/Leaflet").then((Leaflet) => Leaflet.default)),
+  { keepAlive: true }
+);
+
+const cesiumFromPromise = computed(
+  () =>
+    fromPromise(import("../Models/Cesium").then((Cesium) => Cesium.default)),
+  { keepAlive: true }
+);
```

Comment:
Moved tv._cesiumPromise & tv._leafletPromise out to module scope. These are still only loaded when needed (although at the moment in `main` there is no separate Cesium bundle).